ValveResourceFormat/CompiledShader/ShaderFile.cs

```diff
@@ -16,6 +16,7 @@ public class ShaderFile : IDisposable
         public const uint LZMA_DELIM = 0x414D5A4C;
         public const int ZSTD_COMPRESSION = 1;
         public const int LZMA_COMPRESSION = 2;
+        public const int UNCOMPRESSED = 3;
```

Comment:
This isn't the actual value in the shader file, is it? If so, uncompressed should be 0 instead, then you can check that compression>0

Reply:
set it to 0, like that ok?